release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -407,16 +418,30 @@ In the following tables, features are marked with the following statuses:
 
 [id="ocp-4-16-removed-features"]
 === Removed features
-//.APIs removed from Kubernetes 1.27
-//[cols="2,2,2",options="header",]
-//|===
-//|Resource |Removed API |Migrate to
 
-//|`CSIStorageCapacity`
-//|`storage.k8s.io/v1beta1`
-//|`storage.k8s.io/v1`
+[id="ocp-4-16-removed-kube-1-29-apis"]
+==== Beta APIs removed from Kubernetes 1.29
+
+Kubernetes 1.29 removed the following deprecated API, so you must migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-29[Kubernetes documentation].
+
+.APIs removed from Kubernetes 1.29
+[cols="2,2,2,1",options="header",]
+|===
+|Resource |Removed API |Migrate to |Notable changes
+
+|`FlowSchema`
+|`flowcontrol.apiserver.k8s.io/v1beta2`
+|`flowcontrol.apiserver.k8s.io/v1` or `flowcontrol.apiserver.k8s.io/v1beta3`
+|No
```

Comment:
Should this be "None"?

Reply:
Could make an argument for that, but it's consistent with how we've been writing Yes and No for these items for the last few releases, so I'll probably leave it be for now.